includes/sensei-functions.php

```diff
@@ -282,3 +282,20 @@ function sensei_does_theme_support_templates() {
 
 	return in_array( $current_theme, $themes, true ) || current_theme_supports( 'sensei' );
 }
+
+/**
+ * Sorts lessons on edit.php?type=lesson default page
+ * @param type $query
+ */
+function sensei_admin_sort_lessons( $query ) {
+  if( !is_admin() )
+      return;
+
+  $screen = get_current_screen();
+  if( 'edit' == $screen->base && 'lesson' == $screen->post_type && !isset( $_GET['orderby'] ) ) {
+      $query->set( 'orderby', 'meta_value' );
+      $query->set( 'order', '_lesson_order' );
+  }
+  return;
```

Comment:
I don't think you need the `return` here since `pre_get_posts` doesn't need to return a value.